network/netplay/netplay.c

```diff
@@ -1270,6 +1356,10 @@ void netplay_load_savestate(netplay_t *netplay, retro_ctx_serialize_info_t *seri
       netplay->other_frame_count = netplay->self_frame_count;
    }
 
+   /* If we can't send it to the peer, loading a state was a bad idea */
```

Comment:
Shouldn't that be 'sending' and 'is'? If it actually is loading/was, shouldn't that be detected before actually loading that state?